recipe/conda_build_config.yaml

```
@@ -96,6 +96,7 @@ target_goos:
   - windows                    # [win]
 target_goarch:
   - amd64                      # [x86_64]
+  - arm64                      # [arm64 or aarch64]
```

Can you add ppc64le too?

---

I think this is right, but please correct if it is not.  Based on https://github.com/golang/go/commit/09d92b6bbf26f195a1fe136513238d227bc73f82#diff-b3cf214d492e044245cea755cade6d84f10f6b616ef1df51ffb8508088872cbaR11

```suggestion
  - arm64                      # [arm64 or aarch64]
  - ppc64le                   # [ppc64le]
```